test/graph_gen_transfer_tests.cc

```diff
@@ -16,26 +14,15 @@ TEST_CASE("distributed_graph_generator generates required data transfer commands
 	const range<1> test_range = {256};
 	auto buf = dctx.create_buffer(test_range);
 
-	auto rm = [](chunk<1> chnk) -> subrange<1> {
-		switch(chnk.offset[0]) {
-		case 0: return chnk;
-		case 64: return {128, 64};
-		case 128: return {64, 64};
-		case 192: return chnk;
-		default: FAIL("Unexpected offset");
-		}
-		return {};
-	};
```

Comment:
This does not fulfill the range-mapper property (union of chunks must access the union of subranges), so I replaced it with a valid range mapper that would map `i -> n-1-i`. Please review if this test is still meaningful after the change.

Reply:
I guess the idea was that not all nodes have "required data transfers", but I don't see how we could reproduce that behavior with a legal range mapper right now. Also I'm pretty sure we have those code paths covered in other tests anyway, so I think it should be fine.